composer.json

```diff
@@ -22,7 +22,7 @@
     "source": "https://github.com/momentohq/laravel-cache"
   },
   "require": {
-    "momentohq/client-sdk-php": "1.3.0"
+    "momentohq/client-sdk-php": "v1.7.1"
```

Comment:
i've been confused as to why these dep versions are switching between not having a `v` prefix and having a `v` prefix. do you know why?

Reply:
I think git prefers using v for version number from now on: https://www.flownative.com/en/blog/semantic-versioning-and-different-version-number-formats.html#:~:text=Git,common%20way%20to%20denote%20versions.

Reply:
so does it support both, and we are just standardizing on the one that we think is now "best practice"? or did something change about our release process such that it used to require no `v` and now requires `v`?

Reply:
It supports both, the composer does not break if we do not provide a 'v' prefix to our version number.

Reply:
It supports both, the composer does not break if we do not provide a 'v' prefix to our version number.